tutoraspects/plugin.py

```diff
@@ -486,9 +486,8 @@
         )  # pylint: disable=no-member
 except AttributeError as e:
     for service, template_path, priority in MY_INIT_TASKS:
-        full_path: str = pkg_resources.resource_filename(
-            "tutoraspects", os.path.join("templates", *template_path)
-        )
+        templates_path = str(importlib_resources.files("tutoraspects") / "templates")
```

Comment:
nit: can be defined outside of the for loop as `TEMPLATES_PATH`

Reply:
I'm not worried about this one, I'd just like to get a build out